send-message.js

```diff
@@ -1,26 +1,20 @@
-const { parse } = require('node-html-parser');
+const { RichEmbed } = require('discord.js');
 
 const sendMessage = (channel, entry) => {
-  const HREF_LINK = /href="(.*)"/;
-  const IS_URL = /http/;
-  const links = parse(entry.content)
-    .firstChild
-    .querySelectorAll('a')
-    .reduce((accumulator, element) => {
-      const attrs = element.rawAttrs.split(' ');
-      attrs.forEach((attr) => {
-        const matches = attr.match(HREF_LINK);
-        if (matches) {
-          const url = matches[1];
-          if (IS_URL.test(url)) accumulator.push(matches[1]);
-        }
-      });
-      return accumulator;
-    },
-    [])
-    .slice(0,3);
+  const MAX_EMBED_LEN = 5500;
```

Comment:
Discord's richembed max length is 6000, 1024 for field. https://discordapp.com/developers/docs/resources/channel#embed-limits
Lazy implementation that's 'good enough' for now